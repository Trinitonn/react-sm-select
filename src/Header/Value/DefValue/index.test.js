import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import { DefValue } from './index';

describe('DefValue', () => {
  it('should render properly', () => {
    const wrapper = shallow(
      <DefValue
        value={['one', 'two']}
        options={[{ value: 'one', label: 'One' }, { value: 'two', label: 'Two' }]}
      />
    );

    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
