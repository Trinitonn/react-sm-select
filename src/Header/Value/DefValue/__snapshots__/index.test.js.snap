// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DefValue should render properly 1`] = `
<span>
  One, Two
</span>
`;
